homeassistant/components/nintendo_parental/entity.py

```diff
@@ -0,0 +1,40 @@
+"""Base entity definition for Nintendo Parental."""
+
+from __future__ import annotations
+
+from pynintendoparental import Device
+
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import NintendoUpdateCoordinator
+
+
+class NintendoDevice(CoordinatorEntity):
```

Comment:
```suggestion
class NintendoDevice(CoordinatorEntity[..]):
```
Add the coordinator type in the slot